### ISSUE TITLE
`y` key when combined with `f` or `t` doesn't copy correctly

### ISSUE THREAD

**Issue by jrmiller82** [VSCodeVim/Vim#7688]
Expected behaviour:

I want to use the `y` and follow with the grammar `f` `[target char]` to copy from cursor to the target char. 

I.e.

`This sentence I don't want but I want to grab the whole parenthetical following: (this to be copied including the parentheses)`

If you're starting at the first capital T in the foregoing sentence, you'd type `f(` to get to the parentheses, then in normal vim you'd type `yf)` to copy the whole parenthetical. This does nothing; and makes using the macro engine quite hard.

**Comment by jrmiller82** [comment]
Looks like it's the `vim.overrideCopy` command that needs to be set to `true` to get the correct behavior. 

Either the documentation needs to be more explicit, or the `vim.overrideCopy` needs to default to `true` to avoid this silliness. (Who uses vim keys but doesn't do *basic* copying this way?)

**Comment by J-Fields** [comment]
I agree that the docs should be clearer - they've been neglected for a while.

I'm also sympathetic to this being default behavior, but the project's original creators wanted it to match Vim's out-of-the-box experience, and changing the default would be a breaking change at this point. At some point I'll likely make a handful of these sensible breaking changes, but doing so deserves proper planning and communication, which I haven't yet had the time & energy to do.

**Comment by jrmiller82** [comment]
The desired behavior I was describing *is* the out of the box Vim behavior. The current default is not the expected behavior for a Vim user.

**Comment by J-Fields** [comment]
Sorry - I'd skimmed the OP and came to a bad conclusion.

I agree `yf{char}` should work regardless of settings - as far as I can tell, this *does* work regardless of `vim.overrideCopy`. Could you provide your full `vim.*` config?
